Index: src/Receptionist/View/DashBoardForm.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Receptionist/View/DashBoardForm.fxml	(date 1630835646627)
+++ src/Receptionist/View/DashBoardForm.fxml	(date 1630835646627)
@@ -372,9 +372,9 @@
          </styleClass>
          <children>
             <TableView layoutX="8.0" layoutY="6.0" prefHeight="221.0" prefWidth="880.0" stylesheets="@../../CSS/StyleSheet.css">
-              <columns>
-                <TableColumn fx:id="colUserID" prefWidth="74.0" text="User ID" />
-                <TableColumn fx:id="colNIC" prefWidth="108.99996948242188" text="NIC" />
+               <columns>
+                  <TableColumn fx:id="colUserID" prefWidth="74.0" text="User ID" />
+                  <TableColumn fx:id="colNIC" prefWidth="108.99996948242188" text="NIC" />
                   <TableColumn fx:id="COLName" prefWidth="151.0" text="Name" />
                   <TableColumn fx:id="colContactNo" prefWidth="101.0" text="Contact No" />
                   <TableColumn fx:id="colAge" prefWidth="56.0" text="Age" />
@@ -382,7 +382,7 @@
                   <TableColumn fx:id="colReson" prefWidth="142.0" text="Reson" />
                   <TableColumn fx:id="colDoctoerId" text="Doctor Id" />
                   <TableColumn fx:id="colAmount" prefWidth="90.0" text="Amount" />
-              </columns>
+               </columns>
             </TableView>
             <Button layoutX="787.0" layoutY="228.0" mnemonicParsing="false" onAction="#mormalTableRemoveOnAction" prefHeight="30.0" prefWidth="100.0" stylesheets="@../../CSS/StyleSheet.css" text="  Remove">
                <font>
